src/types.hpp

```diff
@@ -59,6 +59,12 @@ struct Action {
     float headAngle;
 };
 
+struct DeltaAction {
+    float dx;
+    float dy;
+    float dyaw;
+};
+
```

Comment:
Lets have a combined Action struct. I think Unions should be supported in madrona. Perhaps we can have somthing like this - 
```suggestion
struct ClassicAction {
    // For acceleration based models
    float acceleration;
    float steering;
    float headAngle;
};
struct DeltaAction {
    // Delta model 
    float dx;
    float dy;
    float dyaw;
};
union Action{
    ClassicAction classicAction;
    DeltaAction deltaAction;
};
```

Reply:
Got it!